sdk/src/main/java/com/bugsnag/android/ndk/NativeBridge.java

```diff
@@ -271,8 +274,16 @@ private boolean is32bit() {
     private void handleAddBreadcrumb(Object arg) {
         if (arg instanceof Breadcrumb) {
             Breadcrumb crumb = (Breadcrumb) arg;
+            Map<String, String> metadata = new HashMap<String, String>();
```

Comment:
This can be replaced with the diamond type since 1.6 for conciseness: `Map<String, String> metadata = new HashMap<>()`